awscli/customizations/cloudformation/artifact_exporter.py

```diff
@@ -432,6 +445,21 @@ def __init__(self, template_path, parent_dir, uploader,
         self.resources_to_export = resources_to_export
         self.uploader = uploader
 
+    def export_global_artifacts(self, template_dict):
+        """
+        Template params such as AWS::Include transforms are not specific to 
+        any resource type but contain artifacts that should be exported,
+        here we iterate through the template dict and export params with a 
+        handler defined in GLOBAL_EXPORT_DICT
+        """
+        for key, val in iter(template_dict.items()):
```

Comment:
Just a python thing. We can simplify this to:
```suggestion
        for key, val in template_dict.items():
```